### PR TITLE
Update readme of KM 

### PR DESCRIPTION
**Pull request by ZiyueXu77**
Fixes # .

### Description

add a "basics of KM" section with real examples to help beginners to understand the method

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/advanced/kaplan-meier-he/README.md [hunk 1]
```diff
@@ -4,16 +4,39 @@ This example illustrates two features:
...
+
+The Kaplan-Meier estimator takes into account the time of the event (e.g. "Survival Days") and whether the event was observed or censored. An event is observed if the event of interest (e.g. "death") occurred at the end of the observation process. An event is censored if the event of interest did not occur (i.e. patient is still alive) at the end of the observation process.
+
+One example dataset used here for Kaplan-Meier analysis is the `veterans_lung_cancer` dataset. This dataset contains information about the survival time of veterans with advanced lung cancer. Below we provide some samples of the dataset:
+
+| ID | Age | Celltype | Karnofsky | Diagtime | Prior | Treat    | Status | Survival Days |
+|----|-----|----------|-----------|----------|-------|----------|--------|--------------|
+| 1  | 64  | squamous | 70        | 5        | yes   | standard | TRUE   | 411          |
+| 20 | 55  | smallcell |  40        |3  |no|standard| FALSE  |123|
+|45|61|adeno|20|19|yes|standard| TRUE   |8|
```
[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/advanced/kaplan-meier-he/README.md]: fixed

**Comment by ZiyueXu77** [comment]
> @ZiyueXu77 also, @SYangster is going to rename WFController to something else

sure, will update the term

**Comment by ZiyueXu77** [comment]
@SYangster, let me know when we finalized the new name for WFController :)

**Comment by YuanTingHsieh** [comment]
Can wait after this https://github.com/NVIDIA/NVFlare/pull/2588 is merged

**Comment by ZiyueXu77** [comment]
> Can wait after this #2588 is merged

sounds good, will make updates after 2588

**Pull request merged**